alpha/apps/kaltura/modules/extwidget/actions/embedIframeJsAction.class.php

```diff
@@ -187,13 +187,20 @@ private function redirectToV7($v7Id, $v2UiConfId, $partnerId, $shouldTranslatePl
 			return;
 		}
 
+		$targetId='';
+		//In v2 'playerId' turns to 'targetId' in v7
+		if($this->getRequestParameter('autoembed'))
+		{
+			$targetId = '&targetId=' . $this->getRequestParameter('playerId');
```

Comment:
can $this->getRequestParameter('playerId') be empty ? if so then we want $targetId=''

Reply:
if its empty then we do not what targetId to be anything, no key nor value

Reply:
if it's empty the code will do
$targetId = '&targetId='
and not 
$targetId=''
as needed